### ISSUE TITLE
Game Crashes at GetName function

### ISSUE THREAD

**Issue by Lessyzz** [sneakyevil/IL2CPP_Resolver#49]
### What is the game you are using with IL2CPP Resolver?

My Own Game

### Details:

When I run this code
The game crashes when the GetName function runs any idea?

### Minimal, Complete and Verifiable Example code:

```
Unity::il2cppArray<Unity::CGameObject*>* m_pObjects = Unity::Object::FindObjectsOfType<Unity::CGameObject>(UNITY_GAMEOBJECT_CLASS);

    for (uintptr_t u = 0U; m_pObjects->m_uMaxLength > u; ++u)
    {
        Unity::CGameObject* m_pObject = m_pObjects->m_pValues[u];
        if (!m_pObject) continue;

        cout << m_pObject->GetName();
    }
```

**Comment by extremeblackliu** [comment]
https://github.com/sneakyevil/IL2CPP_Resolver/issues/48#issuecomment-2119261191

**Comment by extremeblackliu** [comment]
closed for duplicate, please talk on #48
